app/src/main/res/layout/gpodnetauth_credentials.xml

```diff
@@ -33,7 +33,11 @@
         android:layout_margin="8dp"
         android:focusable="true"
         android:focusableInTouchMode="true"
-        android:cursorVisible="true"/>
+        android:cursorVisible="true"
+        android:maxLines="1"
```

Comment:
`singleLine` is deprecated. One should instead use `maxLines` to limit the size of the view, and `inputType="text"` (as opposed to `inputType="textMultiLine"`) to indicate that we don't take line breaks.